setup.py

```diff
@@ -33,14 +33,13 @@
     "enforce-typing==1.0.0.post1",
     "json-sempai==0.4.0",
     "eciespy",
-    "eth-brownie",
+    "eth-brownie==1.19.3",
```

Comment:
Please do not modify this version, this will raise dependency conflicts.

Reply:
This is not what is causing dependency conflicts. Look at all previous runs. Those with python 3.11 are downgrading brownie until 1.8 and now it's 1.19.3, the only release where the race conditions are fixed. I can explain in detail in a call. I've tried changing pip, dependency magic and everything but now the main release is broken for all python versions. At least this one is only broken for python 3.11. Thats because vyper 3.7.0, a dependency of brownie, only recently introduceda python 3.11 as a possibility and they haven't released a 3.8, making brownie incompatible with python 3.11

Reply:
We also have to clean and ground all dependencies for dependabot to actually do its job.